app/views/admin/user.scala.html

```diff
@@ -4,10 +4,12 @@
 @import models.audit.AuditTaskTable
 @import models.label.LabelTable
 @import models.mission.MissionTable
+@import models.region.RegionTable
```

Comment:
You don't use either of these added imports anywhere do you? Just a copy/paste error maybe?

Reply:
Good catch! The two imports were just remnants of what I tried earlier. I deleted them, and everything still works.